apache/website.conf

```diff
@@ -17,14 +25,26 @@
         SSLCertificateChainFile /etc/letsencrypt/live/www.berlin.freifunk.net/chain.pem
         SSLCertificateKeyFile   /etc/letsencrypt/live/www.berlin.freifunk.net/privkey.pem
 
-  	Redirect 301 / https://berlin.freifunk.net/
+        RewriteEngine On
+        RewriteCond %{REQUEST_URI} !^/.well-known/
+        RewriteRule ^/$ https://berlin.freifunk.net/de/ [R=301,L]
+        RewriteRule ^/index_en/$ https://berlin.freifunk.net/en/ [R=301,L]
+        RewriteRule ^/network/map/$ https://berlin.freifunk.net/de/map/ [R=301,L]
+        RewriteRule ^/contact/$ https://berlin.freifunk.net/de/contact/ [R=301,L]
+        RewriteRule ^/downloads/$ https://berlin.freifunk.net/de/downloads/ [R=301,L]
+        RewriteRule ^/participate/overview/$ https://berlin.freifunk.net/de/participate/ [R=301,L]
+        RewriteRule ^/participate/howto/$ https://berlin.freifunk.net/de/participate/ [R=301,L]
+        RewriteRule ^/impressum/$ https://berlin.freifunk.net/de/impressum/ [R=301,L]
+
 </VirtualHost>
 
 <VirtualHost *:80>
 	ServerName www.berlin.freifunk.net
    	ServerAdmin "info@berlin.freifunk.net"
 
-  	Redirect 301 / https://berlin.freifunk.net/
+        RewriteEngine On
+        RewriteCond %{REQUEST_URI} !^/.well-known/
+        RewriteRule .* https://berlin.freifunk.net/de/ [R=301,L]
 </VirtualHost>
 
 <VirtualHost *:443>
```

Comment:
Why is there no rewrite set for this block?

Reply:
I don't understand the question. The redirect is changed from a redirect that only matches the root to a redirect that covers the whole host and redirects everything to https://berlin.freifunk.net/de/

Reply:
The github preview might be misleading. I comented line 50. Because this is the only virtual host without a redirect. You set redirects for www.berlin.freifunk.net at 80 and 443 and for berlin.freifunk.net at 80 but not at 443. Maybe that is covered by some rules already, but then I missed that. (I am really quiet new to apache configs, worked with nginx most of the time so far.)

Reply:
ah, verstehe. ist angepasst.